lib/main_common.pm

```diff
@@ -2068,10 +2068,18 @@ sub load_toolchain_tests {
 }
 
 sub load_publiccloud_tests {
-    loadtest "publiccloud/prepare_tools" if get_var('PUBLIC_CLOUD_PREPARE_TOOLS');
-    loadtest "publiccloud/upload_image"  if get_var('PUBLIC_CLOUD_IMAGE_LOCATION');
-    loadtest "publiccloud/ipa"           if get_var('PUBLIC_CLOUD_IPA_TESTS');
-    loadtest 'publiccloud/run_ltp'       if get_var('PUBLIC_CLOUD_LTP');
+    if (get_var('PUBLIC_CLOUD_PREPARE_TOOLS')) {
+        loadtest "publiccloud/prepare_tools";
```

Comment:
not just avoiding to run upload_img in ipa and ltp. In addition you take away ability to execute prepare_tools in scenarios with ipa and ltp. Are sure that it is what we need ? 
@cfconrad

Reply:
as fas as I know, prepare_tools is always executed alone, never together with ipa/ltp...